dhis-2/dhis-support/dhis-support-test/src/main/java/org/hisp/dhis/DhisConvenienceTest.java

```diff
@@ -2315,16 +2315,17 @@ protected User createUserAndInjectSecurityContext( Set<OrganisationUnit> organis
 
         if ( auths != null )
         {
-            authorities.addAll( Lists.newArrayList( auths ) );
+            authorities.addAll( asList( auths ) );
         }
 
+        char uniqueCharacter = nextUserName++;
         UserAuthorityGroup group = new UserAuthorityGroup();
-        group.setName( "Superuser" );
+        group.setName( "Superuser_" + uniqueCharacter );
```

Comment:
This is needed so the method can be called multiple times in a test. Otherwise the group name used would not be unique.